src/SarifCli/JPointerStringExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.CodeAnalysis.Sarif.Cli
+{
+    // TODO: These methods belong in the Microsoft.Json.Pointer library.
```

Comment:
Fixed.

---
In reply to: [77239499](https://github.com/Microsoft/sarif-sdk/pull/501#discussion_r77239499) [](ancestors = 77239499,77238851)